serverlessworkflow/antora-playbook.yml

```diff
@@ -0,0 +1,20 @@
+site:
```

Comment:
One note about the file organization: if you suspect there may be other components than `serverlessworkflow` in the future, I would move the descriptors of the website (`antora-playbook*.yml`) and the entire website assembly (the `antora` folder) outside this directory.

In other words, I would keep the component content and the website assembly apart.

Reply:
Hey @rsynek many thanks for reviewing this PR! Yea, we will have more components within this same repo.

Reply:
Hi @rsynek I have changed the file structure to have multiple components now.